crates/proof-of-sql-parser/src/intermediate_ast.rs

```diff
@@ -224,6 +224,7 @@ impl Expression {
             expr: Box::new(self),
         })
     }
+    // TODO: add panic docs
     /// Create an `AliasedResultExpr` from an `Expression` using the provided alias.
```

Comment:
Something like this should allow the lints to pass.
```suggestion
    /// Create an `AliasedResultExpr` from an `Expression` using the provided alias.
    ///
    /// # Panics
    ///
    /// TODO: add panic docs
```